Limber/MIXMockServer.cpp

```diff
@@ -95,6 +130,20 @@ class Server {
             }
             stanza->setFrom(session->getRemoteJID());
 
+            auto i = participantInformationMap_.find(session->getRemoteJID().toBare());
```

Comment:
I've now passed stanza to corresponding handler functions i.e. `handleElementRecievedForMIXChannel`, `handleElementRecievedForAccount` and `handleElementRecievedForMIXService`. 

However retrieving / setting of `information_` field in `handleElementReceived` is just either creating a new `MIXParticipantInformation` object for a new user or retrieving an already stored object for current user. Therefore, it is better to store `information_` here, rather than checking `participantInformationMap_` has current user JID in every handler function and then retrieving a `information_` instance.